context/app/static/js/pages/Dataset/hooks.ts

```diff
@@ -62,58 +81,77 @@ function useProcessedDatasets() {
     query: {
       bool: {
         // TODO: Futher narrow once we understand EPICs.
-        must: [getIDsQuery(descendant_ids), excludeSupportEntitiesClause, excludeComponentDatasetsClause],
+        must: [getIDsQuery(descendant_ids), excludeComponentDatasetsClause],
       },
     },
-    _source: ['hubmap_id', 'entity_type', 'uuid', 'assay_display_name', 'files', 'pipeline'],
+    _source: [
+      'hubmap_id',
+      'entity_type',
+      'uuid',
+      'assay_display_name',
+      'files',
+      'pipeline',
+      'status',
+      'metadata',
+      'creation_action',
+      'created_timestamp',
+    ],
     size: 10000,
   };
 
   const isPrimary = processing === 'raw';
 
   const shouldFetch = isPrimary && entityIsDataset;
 
-  const { searchHits, isLoading } = useSearchHits<ProcessedDatasetTypes>(query, {
+  const { searchHits, isLoading } = useSearchHits<ProcessedDatasetInfo>(query, {
     shouldFetch,
   });
 
-  const { data: confs, isLoading: isLoadingConfs } = useVitessceConfs({ uuids: descendant_ids, shouldFetch });
-
-  return { searchHits, confs, isLoading: isLoading || isLoadingConfs };
+  return { searchHits, isLoading };
 }
 
 function getProcessedDatasetSection({
   hit,
   conf,
 }: {
-  hit: Required<SearchHit<ProcessedDatasetTypes>>;
+  hit: Required<SearchHit<ProcessedDatasetInfo>>;
   conf?: VitessceConf;
 }) {
-  const { pipeline, hubmap_id } = hit._source;
+  const { pipeline, hubmap_id, files, metadata } = hit._source;
 
   const shouldDisplaySection = {
     summary: true,
-    visualization: Boolean(conf),
-    files: Boolean(hit?._source?.files),
+    visualization: Boolean(conf && 'data' in conf && conf?.data),
+    files: Boolean(files),
+    analysis: Boolean(metadata?.dag_provenance_list),
   };
 
   const sectionsToDisplay = Object.entries(shouldDisplaySection).filter(([_k, v]) => v === true);
 
   return {
     // TODO: Improve the lookup for descendants to exclude anything with a missing pipeline name
-    ...getSectionFromString(pipeline ?? hubmap_id, `${hubmap_id}-section`),
-    items: sectionsToDisplay.map(([s]) => ({ ...getSectionFromString(s), hash: `${s}-${hubmap_id}` })),
+    ...getSectionFromString(pipeline ?? hubmap_id, datasetSectionId(hit._source, 'section')),
```

Comment:
In this scenario - Search API responses return field-level data inside a `_source` field for each hit, since each hit also includes `_id` (the entity's UUID), `_score` (how closely the entity matches query parameters if using looser matching than `must`), `_index` (which search index the response came from, a constant value depending on the environment in our usecase), and `_type` (`"doc"` in our case).

In non-search API contexts, including the `_` prefix for `_first` indicates that the variable is not used, and keeps ESLint from yelling at us for unused variables. For example, if you want to drop the first item in a list, you can destructure it like so:

```
const [_first, rest] = [1, 2, 3, 4, 5]
```

Reply:
Ahh got it! Thanks for the explanation.